pitch.md

```diff
@@ -1,8 +1,17 @@
-# Project Title (replace me)
+# Construct
 
 ## Overview
 _What's this project about, briefly?_
 
+The Construct language is a domain specific language for doing geometric
+constructions on the fly.
```

Comment:
In the rest of your writeup, you seem to be considering mostly 2 dimensional constructions. Would the language be concerned exclusively with 2D figures, or do you want it to be able to handle 3D as well. Alternatively, how might you make the language capable of being _extended_ (maybe by someone else) to support 3D figures? I think if you could make a language for 2D constructions that's extensible to 3 (or even arbitrary?) dimensions that could be considered a victory in language design.